src/batched/KokkosBatched_Gemm_Decl.hpp

```diff
@@ -95,118 +146,125 @@ struct BatchedSerialGemm;
 #if defined(KOKKOS_ENABLE_HIP)
-    constexpr bool on_gpu = std::is_same<typename CViewType::execution_space,
-                                         Kokkos::Experimental::HIP>::value;
+  constexpr bool on_gpu = std::is_same<typename CViewType::execution_space,
+                                       Kokkos::Experimental::HIP>::value;
 #endif  // KOKKOS_ENABLE_HIP
 
 #if !defined(KOKKOS_ENABLE_CUDA) && !defined(KOKKOS_ENABLE_HIP)
-    constexpr bool on_gpu = false;
+  constexpr bool on_gpu = false;
 #endif
 
 #if __x86_64__
-    constexpr bool on_intel =
-        std::is_same<typename CViewType::execution_space::memory_space,
-                     Kokkos::HostSpace>::value;
+  constexpr bool on_intel =
+      std::is_same<typename CViewType::execution_space::memory_space,
+                   Kokkos::HostSpace>::value;
 #else
-    constexpr bool on_intel = false;
+  constexpr bool on_intel = false;
 #endif  // Intel architectures
 
 #if defined(__ARM_ARCH_ISA_A64)
-    constexpr bool on_a64fx =
-        std::is_same<typename CViewType::execution_space::memory_space,
-                     Kokkos::HostSpace>::value;
+  constexpr bool on_a64fx =
+      std::is_same<typename CViewType::execution_space::memory_space,
+                   Kokkos::HostSpace>::value;
 #else
-    constexpr bool on_a64fx = false;
+  constexpr bool on_a64fx = false;
 #endif
 
-    // Select whether to calculate a rank-0 or rank-2 result per thread
-    using resultsPerThread =
-        typename std::conditional<!is_vector && on_gpu, ResultsPerThread::Rank0,
-                                  ResultsPerThread::Rank2>::type;
-
-    // Selects whether to use the register blocking or non-register blocking
-    // implementation of SerialGemm.
-    using mode_type = typename std::conditional<
-        is_vector,
-        typename std::conditional<on_gpu || on_intel, Algo::Gemm::Blocked,
-                                  Algo::Gemm::Unblocked>::type,
-        typename std::conditional<
-            on_gpu, Algo::Gemm::Unblocked,
-            typename std::conditional<on_a64fx, Algo::Gemm::Unblocked,
-                                      Algo::Gemm::Blocked>::type>::type>::type;
+  // Select whether to calculate a rank-0 or rank-2 result per thread
+  using resultsPerThread =
+      typename std::conditional<!is_vector && on_gpu, ResultsPerThread::Rank0,
+                                ResultsPerThread::Rank2>::type;
+
+  // Selects whether to use the register blocking or non-register blocking
+  // implementation of SerialGemm.
+  using mode_type = typename std::conditional<
+      is_vector,
+      typename std::conditional<on_gpu || on_intel, Algo::Gemm::Blocked,
+                                Algo::Gemm::Unblocked>::type,
+      typename std::conditional<
+          on_gpu, Algo::Gemm::Unblocked,
+          typename std::conditional<on_a64fx, Algo::Gemm::Unblocked,
+                                    Algo::Gemm::Blocked>::type>::type>::type;
 
 #if 0
```

Comment:
Should this be made into a verbosity controled output for debugging or to give extra info to the user if required?